sample-apps/javascript-sample-app/server.js

```diff
@@ -0,0 +1,117 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS'" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ */
+
+'use strict';
+
+// setting up metrics
+// const my_meter = require('./create-a-meter');
+// const { emitsPayloadMetric, emitReturnTimeMetric } = require('./get-meter-emit-functions')(my_meter)
+
+// TracerProvider must be initialized before instrumented packages
+// (i.e. before 'aws-sdk' and 'http') are imported.
+
+// setting up the traces and metrics
+const tracer = require('./tracer');
+const meter = require('./meter');
+
+const http = require('http');
+const AWS = require('aws-sdk');
+const Worker = require('worker_threads');
+
+const { updateTotalBytesSent, updateLatencyTime } = require('./request-metrics')(meter);
+
+const api = require('@opentelemetry/api');
+const create_cfg = require('./config');
+
+function startServer(file='./config.yaml') {
+    const worker = new Worker.Worker('./random-metrics.js');
+    const cfg = create_cfg.create_config(file);
+    const server = http.createServer(handleRequest);
+    server.listen(cfg.Port, cfg.Host, (err) => {
+      if (err) {
+        throw err;
+      }
+      console.log(`Node HTTP listening on ${cfg.Host}:${cfg.Port}`);
+    });
+}
+
+function handleRequest(req, res) {
+    const cfg = create_cfg.create_config('./config.yaml');
+    const requestStartTime = new Date().getMilliseconds();
+    try {
+        if (req.url === '/') {
```

Comment:
nit, instead of using if else, maybe try to use a map to simplify the code a bit?

```js
routeMapper = {
  "/": (req, res) => { 
        res.end('OK.');
  },
  "/aws-sdk-call": (req, res) => { 
      const s3 = new AWS.S3();
     }
     ....
}

routeMapper[req.url](req, res);
```

Reply:
Fixed